api/src/main/scala/org/ensime/api/outgoing.scala

```diff
@@ -235,6 +235,14 @@ final case class EmptySourcePosition() extends SourcePosition
 final case class OffsetSourcePosition(file: EnsimeFile, offset: Int) extends SourcePosition
 final case class LineSourcePosition(file: EnsimeFile, line: Int) extends SourcePosition
 
+case class SourcePositions(positions: List[SourcePosition]) extends RpcResponse
+
+sealed trait HierarchyInfo
+final case class ClassInfo(fqn: String, sourcePosition: Option[SourcePosition]) extends HierarchyInfo
+final case class TreeInfo(classInfo: ClassInfo, hierarchy: Seq[HierarchyInfo]) extends HierarchyInfo
+
+final case class SymbolTreeInfo(ancestors: Option[HierarchyInfo], inheritors: Option[HierarchyInfo]) extends RpcResponse
```

Comment:
I think the members are available in the graph. I'm surprised that the typeargs and params are not in the graph.